sim/common/cata/other_effects.go

```diff
@@ -825,6 +836,253 @@ func init() {
 			})
 		})
 	}
+
+	for version, _ := range []ItemVersion{ItemVersionLFR, ItemVersionNormal, ItemVersionHeroic} {
+		labelSuffix := []string{" (LFR)", "", " (Heroic)"}[version]
+
+		vialItemID := []int32{77979, 77207, 77999}[version]
+		core.NewItemEffect(vialItemID, func(agent core.Agent) {
+			character := agent.GetCharacter()
+
+			actionID := core.ActionID{SpellID: []int32{109721, 107994, 109724}[version]}
+			minDmg := []float64{3568, 4028, 4546}[version]
+			maxDmg := []float64{5353, 6042, 6819}[version]
+
+			lightningStrike := character.RegisterSpell(core.SpellConfig{
+				ActionID:    actionID,
+				SpellSchool: core.SpellSchoolPhysical,
+				ProcMask:    core.ProcMaskEmpty,
+				Flags:       core.SpellFlagPassiveSpell,
+
+				Cast: core.CastConfig{
+					DefaultCast: core.Cast{
+						NonEmpty: true,
+					},
+				},
+
+				DamageMultiplier: 1,
+				CritMultiplier:   character.DefaultMeleeCritMultiplier(),
+				ThreatMultiplier: 1,
+
+				ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+					spell.CalcAndDealDamage(sim, target, sim.Roll(minDmg, maxDmg), spell.OutcomeMeleeSpecialCritOnly)
+				},
+			})
+
+			core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
+				Name:       "Vial of Shadows Trigger" + labelSuffix,
+				ActionID:   core.ActionID{ItemID: vialItemID},
+				Callback:   core.CallbackOnSpellHitDealt,
+				ProcMask:   core.ProcMaskMeleeOrRanged | core.ProcMaskProc,
+				Outcome:    core.OutcomeLanded,
+				Harmful:    true,
+				ProcChance: 0.45,
+				ICD:        time.Second * 9,
+				Handler: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+					lightningStrike.Cast(sim, result.Target)
+				},
+			})
+		})
+
+		fetishItemID := []int32{77982, 77210, 78002}[version]
+		core.NewItemEffect(fetishItemID, func(agent core.Agent) {
```

Comment:
Maybe add a note to the DBC value not existing and the patch changing it?

Reply:
Are you referring to Vial of Shadows? Yea I guess we could add a comment pointing to the patch notes, although we haven't tracked patch changes for other items afaik :D

Reply:
Ah yes sorry :D Since just like with VPLC we aren't 100% sure about the state this is implemented in before PTR